### ISSUE TITLE
New license request: BALv5 [SPDX-Online-Tools]

### ISSUE THREAD

**Issue by anoraktrend** [spdx/license-list-XML#2577]
**1.** License Name: Berkeley Artistic License V5
**2.** Short identifier: BALv5
**3.** License Author or steward: Lucy Brown
**4.** Comments: This License was made as a means of creating a more legible, correct, and up to date variation Artistic License V1, that has been conformed to my personal interpretation of the BSD, (Berkeley), style. It differs from BSD and Artistic v2 in that it is not re-licensable and it is copyleft rather than permissive. It differs from GPL in that it's short, Clear, and accessible and it contains Metadata to credit developers within the license. While it hasn’t yet been submitted to the OSI and FSF for review, this is my top priority once I have an identifier for ease of reference.
**5.** License Request Url: http://tools.spdx.org/app/license_requests/397
**6.** URL(s): https://raw.githubusercontent.com/anoraktrend/Berkeley-Artistic/refs/heads/main/The%20Berkeley%20Artistic%20license
**7.** OSI Status: Not Submitted
**8.** Example Projects: https://github.com/anoraktrend/frankensrc

**Comment by jlovejoy** [comment]
@anoraktrend - It sounds like this is a newly-drafted license that is not in use yet, is that correct?
If so, it won't meet the "substantial use" criteria for SPDX License List inclusion. 
If you plan to submit it to the OSI, it would be better to do that first as that may result in some changes to the text. (If it is accepted by OSI at whatever point, it is automatically be eligible for inclusion by SPDX.

Keep in mind that "license proliferation" is not seen as beneficial for the open source ecosystem, so you might want to consider if this new license is really needed. That is (also) something that is better suited for discussion with the OSI.

As such, I'll close this issue shortly and you can certainly re-submit later, pending the OSI conversation!

**Comment by anoraktrend** [comment]
It’s most certainly in use by myself at the very least. As mentioned, it’s a modernization of the Artistic V1 license with some changes to fit the BSD style and this version is one that doesn’t violate the OSD. The license is intentionally copyleft as well. I am unsure as to anyone else who uses it at this point in time.

**Comment by richardfontana** [comment]
This license has recently been discussed on threads on OSI's license-review, e.g.:
https://lists.opensource.org/pipermail/license-review_lists.opensource.org/2024-October/005564.html

**Comment by jlovejoy** [comment]
given this is currently under review by OSI, I"m going to mark it as waiting on submitter (even tho it's really waiting on OSI and submitter) and move to the next release.

If the OSI accepts it, it's an automatic inclusion on the SPDX License List.
If there are changes to the text as a result of OSI review, then we don't want to review until the text is stable. 
If OSI rejects it, then we can consider under SPDX inclusion guidelines, but the substantial use factor will be even more critical.

**Comment by karsten-klein** [comment]
**{metæffekt} Universe**
canonical name: [Berkeley Artistic License](https://github.com/org-metaeffekt/metaeffekt-universe/blob/main/src/main/resources/ae-universe/[b]/README.md#user-content-Berkeley-Artistic-License:~:text=Berkeley%20Artistic%20License)
short name: berkeley-artistic-license
markers: Do Not Promote Marker
category: Berkeley Artistic License
OSI status: _none_

**ScanCode**
matched id: [artistic-2.0](https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/artistic-2.0.LICENSE)
matched id: [isc](https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/isc.LICENSE)
matched id: [paul-mackerras-simplified](https://github.com/nexB/scancode-toolkit/blob/develop/src/licensedcode/data/licenses/paul-mackerras-simplified.LICENSE)

**Comment**
We will check whether our representation is correct and matches version 5. Where does this version number originate from? It is not in the text. +1 to add.

**Comment by anoraktrend** [comment]
This is a new license based on previous works. Versions 1-4 weren’t things I looked to make easily accessible, the changes in language between versions were major but I have been procrastinating on getting the license labeled and archived.

**Comment by jlovejoy** [comment]
noting that this has not been accepted by OSI, see https://lists.opensource.org/pipermail/license-review_lists.opensource.org/2025-January/005632.html

**Comment by copernicat** [comment]
@anoraktrend, is this license in use on any projects? I searched GitHub and the only result is this new license request. 
Thank you.

**Comment by copernicat** [comment]
@anoraktrend can you please provide the repos so we can evaluate the SPDX inclusion principle #3 "The license has actual, substantial use such that it is likely to be encountered. Substantial use may be demonstrated via use in many projects, or in one or a few significant projects. For new licenses, there are definitive plans for the license to be used in one or a few significant projects."?
Thanks.

**Comment by anoraktrend** [comment]
I plan to continue to rewrite license to make the license osi compatible, so this can be closed regardless. However, I use it for anoraktrend/bheaded, which is just a fetcher for a few headers from Alpine Linux to install them on any lfs with musl, and anoraktrend/frankensrc, which is a collection of package scripts that are dual licensed. One of these is intended to be an unofficially user package repository for my favorite linux distro, bheaded also is included in this unofficial user package repository.

**Comment by Pizza-Ria** [comment]
It sounds like this is not in signficant use and has been rejected by OSI (with significant negative feedback not just as a redundant set of terms).  Even the submitter seems to agree that this needs revision.  Recommend closing.

**Comment by swinslow** [comment]
Agreed, given that the submitter has indicated this will be rewritten, I am closing this issue. Thank you all!
